lucien-ios/AppDelegate.swift

```diff
@@ -18,7 +19,32 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         self.window = window
         window.rootViewController = RootViewController()
         window.makeKeyAndVisible()
+
+        // Initialize sign-in
+        GIDSignIn.sharedInstance().clientID = "1072472744835-miivpr72vpanvmpm2f3tbb7msae67tii.apps.googleusercontent.com"
+        GIDSignIn.sharedInstance().delegate = self
+
         return true
     }
 
+    private func application(application: UIApplication,
+                             openURL url: URL, options: [UIApplicationOpenURLOptionsKey: AnyObject]) -> Bool {
+        return GIDSignIn.sharedInstance().handle(url as URL!,
+                                                 sourceApplication: options[UIApplicationOpenURLOptionsKey.sourceApplication] as? String,
+                                                 annotation: options[UIApplicationOpenURLOptionsKey.annotation])
+    }
+
+    func sign(_ signIn: GIDSignIn!, didSignInFor user: GIDGoogleUser!, withError error: Error!) {
```

Comment:
Our style guide recommends [grouping protocol conformance methods](https://github.com/IntrepidPursuits/swift-style-guide#code-grouping) with a `// MARK: - <protocol name here>` comment. Makes it easier to read & navigate using the quick navigation bar above the editor.